src/assets/styles/base/_typography.scss

```diff
@@ -1,38 +1,21 @@
-body {
-  color: $body-color;
-  font-family: $font-family-sans-serif-light;
-  font-size: .8em;
-  -webkit-font-smoothing: subpixel-antialiased;
-  letter-spacing: $letter-spacing-base;
-  line-height: $line-height;
-
-  @media screen and (min-width: map-get($bp, "md")) {
-    font-size: .9em;
-  }
-
-  @media screen and (min-width: map-get($bp, "lg")) {
-    font-size: 1em;
-  }
-}
-
 h1,
 h2,
 h3,
 h4,
 h5,
 h6 {
-  color: $heading-color;
-  font-family: $heading-font-family;
-  font-weight: $font-weight-base;
+  color: $color-grey-dark;
+  font-family: $font-family-sans-serif-medium;
+  font-weight: 400;
   line-height: $heading-line-height;
 }
 
 h1 {
   font-family: $font-family-sans-serif-medium;
-  font-size: $font-size-h1-sm;
+  font-size: 2.2rem;
   margin: 0 0 $spacing-3;
 
-  @media screen and (min-width: map-get($bp, "md")) {
+  @media screen and (min-width: map-get($breakpoints, 'breakpoint-3')) {
     font-size: $font-size-h1;
```

Comment:
If we're manually changing a font size, should the smaller screen version not hold the default?